tests/manage/mcg/test_endpoint_autoscale.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from ocs_ci.framework.testlib import MCGTest, tier1, skipif_ocs_version
+from ocs_ci.ocs import constants, defaults, ocp
+
+
+@pytest.mark.polarion_id("OCS-XXXX")
```

Comment:
I will add polarion ids later. Will track it with https://github.com/red-hat-storage/ocs-ci/issues/3121